.ci/scripts/gather_test_models.py

```diff
@@ -90,7 +90,7 @@ def model_should_run_on_event(model: str, event: str) -> bool:
     We put higher priority and fast models to pull request and rest to push.
     """
     if event == "pull_request":
-        return model in ["mv3", "vit"]
+        return model in ["mv3", "vit", "phi4_mini"]  # TODO: remove
```

Comment:
Any reason to remove it, probably it's mostly covered by llama tests?

Reply:
Oh it's just too large to run on every pull request, we only run the small ones on pull